zerver/webhooks/helloworld/view.py

```diff
@@ -5,18 +5,18 @@
 from django.utils.translation import ugettext as _
 
 from zerver.decorator import api_key_only_webhook_view
-from zerver.lib.actions import check_send_stream_message
+from zerver.lib.actions import check_send_webhook_message
 from zerver.lib.request import REQ, has_request_variables
 from zerver.lib.response import json_error, json_success
 from zerver.lib.validator import check_dict, check_string
 from zerver.models import UserProfile
 
 @api_key_only_webhook_view('HelloWorld')
 @has_request_variables
```

Comment:
When we change this, we should also update the integration docs (Which include code from this).